static/js/compose.js

```diff
@@ -886,6 +890,28 @@ exports.needs_subscribe_warning = function (user_id, stream_id) {
     return true;
 };
 
+function insert_webex_video_call_details(details, target_textarea) {
+    const meeting_number_text = i18n.t("Meeting number");
+    const password_text = i18n.t("Password");
+    const start_time_text = i18n.t("Meeting start time");
+    const end_time_text = i18n.t("Meeting end time");
+
+    const start_time_formatted = moment(details.startTime).local().format("llll");
+    const end_time_formatted = moment(details.endTime).local().format("llll");
+
+    compose_ui.insert_syntax_and_focus(
+        `${details.title}
+${meeting_number_text}: ${details.meetingNumber}
+${password_text}: ${details.password}
+${start_time_text}: ${start_time_formatted}
+${end_time_text}: ${end_time_formatted}
+`,
+        target_textarea,
+    );
+
+    insert_video_call_url(details.url, target_textarea);
+}
```

Comment:
Can you post a screenshot of what the resulting message looks like?  I'm curious if we should be using bulleted lists or the like here.

Reply:
![ZulipMeeting](https://user-images.githubusercontent.com/65816804/97283715-14bb8180-1838-11eb-8a1c-d91b248217c0.PNG)